src/components/ImageViewer/index.jsx

```diff
@@ -10,12 +10,24 @@ import './ImageViewer.scss';
 const ImageViewer = props => (
   // If title or caption is an empty string or a string of whitespaces do not display an empty div
   <span id={props.spanId}>
-    <div className="image-border w-100">
-      { (props.title.trim() !== '') ? <div className="image-title">{props.title} </div> : '' }
+    <div className="image-border">
+      { (props.title.trim() !== '') ?
+        <div style={{ maxWidth: `${props.imageWidth}px` }}>
```

Comment:
What's the reason for having a wrapping div instead of setting this on .image-title?